storybook/config/preview.tsx

```diff
@@ -13,12 +14,12 @@ export const argTypes = {
   },
 }
 
-// Set language to Dutch for Canvas and Stories
+// Wrap in Screen, set language to Dutch for Canvas and Stories
 export const decorators = [
   (Story: any) => (
-    <div lang="nl">
+    <Screen lang="nl">
```

Comment:
Should we add an `ams-theme` class to be explicit? The [NLDS Candidate Storybook](https://nl-design-system.github.io/candidate/?path=/story/paragraph--default) does it (see HTML tab) and seems like a good idea.

People can learn (and be instructed) to copy things _inside_ that element, not the wrapper. This is current practice with the `<div lang="nl">` – the new variant `<div class="ams-theme ams-screen" lang="nl">` may make that even clearer.

Reply:
The `lang` attribute is a workaround right? Normally it would go on the html tag.

Also, `Screen` seems to get a default modifier class `ams-screen--wide` where it is a default. So you would not need it, only if you specify x-wide? Which could be a boolean then.

Reply:
> Should we add an ams-theme class to be explicit?

What are we making explicit? [We currently don't use `ams-theme` in Storybook](https://github.com/Amsterdam/design-system/blob/develop/storybook/config/preview.tsx#L1), I don't quite understand why we would want to add an unused class.